lib/routes/home/widgets/home_drawer/widgets/breez_avatar_dialog.dart

```diff
@@ -36,6 +37,7 @@ class BreezAvatarDialogState extends State<BreezAvatarDialog> {
   void initState() {
     super.initState();
     userProfileCubit = context.read<UserProfileCubit>();
+    webhookCubit = context.read<WebhookCubit>();
```

Comment:
Since this cubit is not used on `initState`, we can move it to `build` fn and remove the class variable.

`final WebhookCubit webhookCubit = context.read<WebhookCubit>();`